src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+
+let totalPopulation = 0;
+let count = 0;
+
+populationElements.forEach((element) => {
+  const population = parseInt(element.textContent.replace(/,/g, ''), 10);
+
+  if (!isNaN(population)) {
+    totalPopulation += population;
+    count++;
+  }
+});
+
+const averagePopulation = totalPopulation / count;
```

Comment:
There is a potential issue here if `count` is zero, which would result in a division by zero error. Consider adding a check to ensure `count` is not zero before performing the division.